src/utils/common.js

```diff
@@ -277,16 +277,22 @@ export const commify = (
   return formatedValue;
 };
 
-export const formatFiatValue = (value: number | string, options?: { skipCents?: boolean }): string => {
-  const formatedValue = commify(value, options);
+export const formatFiatValue = (
+  value: number | string,
+  options?: { skipCents?: boolean, isFormatAmount?: boolean },
+): string => {
+  const num = new BigNumber(value).toFixed(2);
+  const formatedValue = options?.skipCents || options?.isFormatAmount ? commify(value, options) : formatBigAmount(num);
```

Comment:
I'm not sure I follow the options :( 
1. `skipCents` and `isFormatAmount` according to ternary check does exactly the same so why keep both?
2. Previously `commify` was not taking `skipCents` into credit, but now it does?

Reply:
@poocart I have added isFormatAmount( name changed to shoudCommify) to prevent the formatting in Detail model and review screen and there skipCents are not true. As per @paragramm in Detail and review screen amount should not be formatted as K,M,B.

Reply:
Sorry Divya, I'm getting little confused here looking at this, I wonder how we can simplify, maybe `isFormatAmount` could be changed to something more reasonable that would make more sense? Since the method itself is calling for `format`  🤔

Reply:
Thanks